samba/rootfs/usr/share/tempio/smb.gtpl

```diff
@@ -26,6 +26,8 @@
    dos charset = CP850
    unix charset = UTF-8
 
+   vfs objects = streams_xattr
```

Comment:
Reading a bit about Samba and macOS interoperability the two modules `fruit` and `catia` come often up. There is an example config in the fruit man page at https://www.samba.org/samba/docs/current/man-html/vfs_fruit.8.html.

I'd say that the default configs for the fruit module are probably fine for our purpose, but maybe it makes sense to enable those additional modules too? :thinking: 

```suggestion
   vfs objects = catia fruit streams_xattr
```